tsconfig.frontend.json

```
@@ -0,0 +1,24 @@
+{
+    "extends": "./tsconfig.base.json",
+    "compilerOptions": {
+        /* Language and Environment */
+        "lib": ["ES2022", "DOM", "DOM.Iterable"],
+        "jsx": "react", // TODO: Egen PR. Bruk react-jsx og fjern all overflødige imports
+
+        /* Emit */
+        "noEmit": true,
+
+        /* Modules */
+        "module": "ESNext",
+        "moduleResolution": "Bundler",
+        "moduleDetection": "force",
+        "noUncheckedSideEffectImports": false, // Trenger for import av @navikt/ds-css
+        "types": [
+            "vite/client",
+            "vitest/globals",
+            "@testing-library/jest-dom"
+        ]
+    },
+    "include": ["src/frontend"],
+    "exclude": ["src/backend"]
```

Burde det være noe slikt? Ser det er gjort i `tsconfig.backend.json`

```suggestion
    "include": ["./src/frontend"],
    "exclude": ["./src/backend"]
```

---

Ja 🤔 Det funker jo, men best å det likt